Thrive.csproj

```diff
@@ -41,6 +41,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <ConsolePause>false</ConsolePause>
+    <LangVersion>latest</LangVersion>
```

Comment:
Is it right that this is added to just this group? Also mono and whatever build tool is used might have different C# versions, so shouldn't an explicit version be used instead?

Reply:
I am not sure, that just how it worked for me

Reply:
I mean this might not work for everyone. So I'd prefer the explicit version setting in every target, or this line to be removed.

Reply:
I think it best for me to just remove it